### ISSUE TITLE
Parallelism for BLIP 2 on a 8 GPUs instance

### ISSUE THREAD

**Issue by rebecasgo** [salesforce/LAVIS#255]
Does LAVIS support parallelism of inference for BLIP 2? 

The idea is to run batch inference on an 8x A100 cluster instance since there is no other option for an A100 GPU instance for Databricks. 

Is there a way to take advantage of the 8 GPUs?

Kind regards,

**Comment by ECE-Engineer** [comment]
if you quantize the model, you could run it on cheaper hardware.
idk if that's what you want tho

**Comment by rebecasgo** [comment]
> if you quantize the model, you could run it on cheaper hardware. idk if that's what you want tho

That would be a good approach if the performance remains better than the smaller version from LAVIS "pretrain_opt2.7b". 

Can the BLIP 2 model be quantized with `torch.ao.quantization`? Will give it a try and update the issue

**Comment by dxli94** [comment]
@rebecasgo, you can load LAVIS model and use whatever distributed learning framework such as pytorch for that purpose.

**Comment by rebecasgo** [comment]
> @rebecasgo, you can load LAVIS model and use whatever distributed learning framework such as pytorch for that purpose.

What are the steps for running LAVIS model locally, without the need of downloading it every time?
